module/VuFind/src/VuFind/Form/Handler/Database.php

```diff
@@ -89,20 +100,25 @@ public function handle(
     ): bool {
         $fields = $form->mapRequestParamsToFieldValues($params->fromPost());
         $fields = array_column($fields, 'value', 'name');
-
+        //Backward compatibility: convert Laminas\Db to Doctrine;
```

Comment:
Minor style point: I would add spaces after the `//` on these two lines -- in VuFind's style, we generally put a space after the comment character(s) for human-readable comments, but we do not put a space when we are commenting out code (e.g. to provide examples). This helps to distinguish one case from the other.